csrc/cutlass_extensions/vllm_numeric_conversion.cuh

```diff
@@ -0,0 +1,538 @@
+// Based off of: cutlass/numeric_conversion.h
+
+#pragma once
+
+#include "cutlass/numeric_conversion.h"
+#include "cutlass_extensions/vllm_custom_types.cuh"
+
+namespace cutlass {
+
+// TODO (Lucas): Implement
+// for Array<cutlass::float8_e4m3fn, N> <= Array<vllm_uint4b8_t, N>
+
+// ....
+
+// for Array<cutlass::half_t, N> <= Array<vllm_uint4b8_t, N>
```

Comment:
nit: I think it'd be more clear as `// For Array<vllm_uint4b8_t, N> to Array<cutlass::half_t, N>`. The order and the `<=` was weird at a glance

Reply:
ya I agree, but thats how its written for the vast majority of the ones in:
https://github.com/NVIDIA/cutlass/blob/cutlass-3.5.0/include/cutlass/numeric_conversion.h
so was favoring consistency here, I still think thats the highest priority